### PR TITLE
page layout for download

### PR DESCRIPTION
**Pull request by djmaluy**
https://trello.com/c/dX78OGL9/204-cv-%D0%B4%D0%B5%D0%B2%D0%B0-%D0%B7-%D0%BC%D0%BE%D0%B6%D0%BB%D0%B8%D0%B2%D1%96%D1%81%D1%82%D1%8E-%D0%B7%D0%B0%D0%B2%D0%B0%D0%BD%D1%82%D0%B0%D0%B6%D1%83%D0%B2%D0%B0%D1%82%D0%B8-pdf

### CODE REVIEW

File: app/models/resume.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Resume < ApplicationRecord
+  has_many :resume_experiences
+  has_many :resume_projects
+  has_many :resume_skills
+  has_many :resume_educations
+  has_many :resume_languages
+  has_many :resume_books
```
Comment (AstraL) [inline app/models/resume.rb:9]: You can make better associations names
`has_many :books, class_name: 'ResumeBooks'`

File: app/models/resume_language.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ResumeLanguage < ApplicationRecord
+  belongs_to :resume
+
+  validates :resume_id, presence: true
```
Comment (AstraL) [inline app/models/resume_language.rb:6]: no need to check resume_id, belongs_to make this if there is optional: true flag

File: config/environments/development.rb [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
   # Show full error reports.
   config.consider_all_requests_local = true
+  Rails.application.routes.default_url_options[:host] = 'localhost:4000'
```
Comment (AstraL) [inline config/environments/development.rb:18]: make this ENV variable, add add similar line to production.rb

File: web/src/hooks/resume/useCreateResumeBook.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import { useGetResume } from './useGetCvs'
+import { IResumeBook } from '../../types/Resume/IResumeBook'
+
+export const useCreateResumeBook = () => {
+  const getUsersResume = useGetResume()
+
+  return (book: IResumeBook, resume_id: number) => {
+    createResumeBook(book)
+      .then(() => {
+        successNotify('Book added')
```
Comment (AstraL) [inline web/src/hooks/resume/useCreateResumeBook.ts:13]: add redux actions here too, it helps to understand what going on

File: web/src/toolkitRedux/toolkitSlice.ts [hunk 5]
```diff
@@ -78,6 +80,9 @@ const toolkitSlice = createSlice({
     loadUserLeaveCounter(state, action) {
       state.leaveCounter = action.payload
     },
+    loadResumes(state, action) {
```
Comment (AstraL) [inline web/src/toolkitRedux/toolkitSlice.ts:83]: check if this method used somewhere, I'll remove old redux store

File: web/src/types/Resume/IResume.ts [hunk 6]
```diff
@@ -1,8 +1,25 @@
...
     summary: string
     user_id: number
+    avatar: string
+    id?: number
+    skills?: IResumeSkill[]
+    country?: string
+    city?: string
+    experiences: IResumeExperience[]
+    resume_projects: IResumeProject[]
+    languages: IResumeLanguage[]
```
Comment (asmlvsk) [inline web/src/types/Resume/IResume.ts:22]: Тобто skills у нас можуть бути андефайнд, а languages ні?

**Pull request merged**